src/parser/postgresparser.cpp

```diff
@@ -1732,13 +1729,96 @@ parser::TransactionStatement *PostgresParser::TransactionTransform(
   }
 }
 
+parser::AlterTableStatement *PostgresParser::AlterTransform(
+    AlterTableStmt *root) {
+  // TODO (shilun) adding alter type check
+  parser::AlterTableStatement *result = new AlterTableStatement(
+      parser::AlterTableStatement::AlterTableType::INVALID);
+
+  // Get database and table name
+  RangeVar *relation = root->relation;
+  result->table_info_ =
+      std::unique_ptr<parser::TableInfo>(new parser::TableInfo());
+  if (relation->relname) {
+    result->table_info_->table_name = strdup(relation->relname);
+  }
+  if (relation->catalogname) {
+    result->table_info_->database_name = strdup(relation->catalogname);
+  }
+
+  for (auto cell = root->cmds->head; cell != NULL; cell = cell->next) {
+    auto cmd = reinterpret_cast<AlterTableCmd *>(cell->data.ptr_value);
+    switch (cmd->subtype) {
```

Comment:
Most cases in the switch clause are commented. Only default case works. Maybe you can delete redundant codes?